### PR TITLE
Add better error handling for problems with no test cases.

### PR DESCRIPTION
**Pull request by radwouters**
When installing DOMJudge in my local environment, I got an SQL error which was difficult to understand. Fortunately, there was Nicky G to help me out. The error is now a lot more understandable!

### CODE REVIEW

File: webapp/src/Service/DOMJudgeService.php [hunk 1]
```diff
@@ -53,6 +53,7 @@
 use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
 use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
 use ZipArchive;
+use function count;
```
Comment (nickygerritsen) [inline webapp/src/Service/DOMJudgeService.php:56]: Do we need this?

Comment (radwouters) [reply webapp/src/Service/DOMJudgeService.php]: No, it was a mixup in my phpstorm config. Fixed.

**Review by nickygerritsen — APPROVED** [review]

**Review by meisterT — APPROVED** [review]

**Comment by thijskh** [comment]
Love this as it’s a common situation that’s now clearly handled.

Maybe a nitpick but is it a BadRequest when the serverside config is incomplete?

**Comment by tuupke** [comment]
Thanks @radwouters! This definitely improves things!

A next step would be to better handle the case of having no testcases. A team should not be presented with a BadRequest  when submitting. They can't change anything to prevent this failure. A different way to handle this situation is to accept the submission but raise an internal error and disable the problem. This should result in the submission being queued until the problem gets enabled again.

We should consider to not allow judging on a problem with zero testcases, and only allow judging when the first testcase gets added. Maybe add a checkbox in the interface so you can decide whether to enable.

**Comment by thijskh** [comment]
@tuupke is right and this is a long standing problem. However I would not get in the way of this concrete improvement over the status quo.

**Comment by meisterT** [comment]
I think we all agree 👍

**Comment by nickygerritsen** [comment]
I propose to merge this as is and then later do the fancy thing. Anyone opposed?

**Comment by meisterT** [comment]
Do it!

**Pull request merged**